modules/dxp/apps/portal-reports-engine-console/portal-reports-engine-console-web/src/main/resources/META-INF/resources/admin/definition/edit_definition.jsp

```diff
@@ -207,6 +207,8 @@ else {
 	</aui:button-row>
 </aui:form>
 
+<script src="../js/main.js" type="text/javascript"></script>
```

Comment:
Hey @wanderlast, could you explain to me why you called the file `../js/main.js` instead of `../js/ReportParameters.js`

Thanks!

Reply:
Hi @igor-franca, it's due to me not noticing the extent of the refactor that occurred in LPS-201366. Liferay.Report was previously instantiated in main.js which was causing the issues mentioned. This may not be necessary in Master anymore, I'll need to test.

Reply:
Hi @igor-franca,

I've tested and could still reproduce on my end, so I've updated the reference to ReportParameters.js. It appears to work properly again now.

Reply:
Hey @wanderlast, I followed the steps in the ticket and I can't reproduce this problem. I am using the Google Chrome browser. Could you send a video of the bug reproduction. Thanks!

Reply:
@igor-franca hm, it appears I can't reproduce either anymore. It's possible I hadn't properly cleared my cache. Since you're also confirming that this is fixed in Master, I'll close this and work on the backport. Thanks!